occurrences/tests/test_notifications.py

```diff
@@ -208,11 +208,21 @@ def test_cancel_occurrence_notification(
     notification_template_cancel_occurrence_en,
     notification_template_cancel_occurrence_fi,
 ):
-    study_groups = StudyGroupFactory.create_batch(3)
-    for s in study_groups:
-        EnrolmentFactory(study_group=s, occurrence=occurrence, person=s.person)
+
+    for status in Enrolment.STATUSES:
+        for s in StudyGroupFactory.create_batch(2):
+            EnrolmentFactory(
+                study_group=s, occurrence=occurrence, person=s.person, status=status
+            )
+    notifiable_enrolments_count = occurrence.enrolments.all().count() - (
+        occurrence.enrolments.filter(status=Enrolment.STATUS_CANCELLED)
+        .filter(status=Enrolment.STATUS_DECLINED)
+        .count()
+    )
```

Comment:
Notification should be sent to contact persons who have not cancelled their enrolment yet or it has not declined.